### ISSUE TITLE
Package errors during mach cargo-update

### ISSUE THREAD

**Issue by frewsxcv** [servo/servo#6287]
```
~/D/r/servo (master|✔) $ ./mach cargo-update -a
components/servo
    Updating registry `https://github.com/rust-lang/crates.io-index`
    Updating git repository `https://github.com/servo/rust-png`
    Updating git repository `https://github.com/servo/rust-geom`
    Updating git repository `https://github.com/servo/rust-stb-image`
    Updating git repository `https://github.com/servo/string-cache`
    Updating git repository `https://github.com/servo/rust-azure`
    Updating git repository `https://github.com/servo/rust-layers`
    Updating git repository `https://github.com/servo/rust-selectors`
    Updating git repository `https://github.com/servo/libfreetype2`
    Updating git repository `https://github.com/servo/rust-core-foundation`
    Updating git repository `https://github.com/servo/rust-freetype`
    Updating git repository `https://github.com/servo/rust-core-text`
    Updating git repository `https://github.com/servo/rust-egl`
    Updating git repository `https://github.com/servo/skia`
    Updating git repository `https://github.com/servo/rust-core-graphics`
no matching package named `core_foundation` found (required by `azure`)
location searched: https://github.com/servo/rust-core-foundation
version required: *
ports/cef
    Updating git repository `https://github.com/servo/gleam`
    Updating git repository `https://github.com/servo/rust-azure`
    Updating git repository `https://github.com/servo/rust-layers`
    Updating git repository `https://github.com/servo/rust-core-text`
    Updating registry `https://github.com/rust-lang/crates.io-index`
    Updating git repository `https://github.com/servo/rust-mozjs`
    Updating git repository `https://github.com/servo/rust-cgl`
    Updating git repository `https://github.com/servo/rust-stb-image`
    Updating git repository `https://github.com/servo/rust-cocoa`
    Updating git repository `https://github.com/servo/rust-core-foundation`
    Updating git repository `https://github.com/servo/rust-geom`
    Updating git repository `https://github.com/servo/rust-core-graphics`
    Updating git repository `https://github.com/servo/rust-png`
no matching package named `core_text` found (required by `embedding`)
location searched: https://github.com/servo/rust-core-text
version required: *
ports/gonk
    Updating git repository `https://github.com/servo/gleam`
    Updating registry `https://github.com/rust-lang/crates.io-index`
    Updating git repository `https://github.com/servo/rust-geom`
    Updating git repository `https://github.com/servo/rust-layers`
    Updating git repository `https://github.com/servo/rust-egl`
    Updating git repository `https://github.com/servo/rust-core-graphics`
    Updating git repository `https://github.com/servo/rust-png`
    Updating git repository `https://github.com/servo/rust-azure`
    Updating git repository `https://github.com/servo/rust-core-text`
    Updating git repository `https://github.com/servo/rust-x11-clipboard`
no matching package named `core_graphics` found (required by `compositing`)
location searched: https://github.com/servo/rust-core-graphics
version required: *
```

**Comment by SimonSapin** [comment]
Some package names have been changed, e.g. https://github.com/servo/core-foundation-rs/commit/fd8bed8a41b244fb2632096f883e140a260d738d. `Cargo.toml` files in Servo need to be updated accordingly.

**Comment by frewsxcv** [comment]
Was just about to open a pull request in this repo to fix them

**Comment by frewsxcv** [comment]
Fixed in #6295
